Dockerfile

```diff
@@ -1,28 +1,32 @@
-FROM node:10-alpine
+FROM node:10-alpine as base
 LABEL Description="Web Frontend of the Social Network Human-Connection.org" Vendor="Human-Connection gGmbH" Version="0.0.1" Maintainer="Human-Connection gGmbH (developer@human-connection.org)"
 
 # Expose the app port
-EXPOSE 3000
-
+ARG BUILD_COMMIT
+ENV BUILD_COMMIT=$BUILD_COMMIT
 ARG WORKDIR=/nitro-web
 RUN mkdir -p $WORKDIR
 WORKDIR $WORKDIR
 
 # See: https://github.com/nodejs/docker-node/pull/367#issuecomment-430807898
 RUN apk --no-cache add git
 
-# Install Web Application
-COPY package.json .
-COPY yarn.lock .
-COPY styleguide/ ./styleguide
-RUN yarn install --production=false --frozen-lockfile --non-interactive
+COPY . .
```

Comment:
To save more space, we should not copy everything into the base but into the build-and-test image, and then copying only the stuff needed. Thats also because the styleguide is quite heavy and I think only the dist directory is needed. 

But that could by done also in another pr.

Reply:
From what I observed entering the `build-and-test` image, we cannot copy over anything that was not first copied over by the base image.